### PR TITLE
Fix up Pulp Smash #301

### PR DESCRIPTION
**Pull request by danuzclaudes**
This commit fixes the Pulp Smash issue #301 and Pulp issue #1938.

Test steps:
    1. Create and sync a repository.
    2. Checkout to content directory /var/lib/pulp/content/units/rpm/.
    3. Remove some random contents in that folder.
    4. Trigger a force sync to verify if the removed contents will be restored.

Test output:

```
$ python -m unittest pulp_smash.tests.rpm.cli.test_sync.ForceSyncTestCase
[sudo] password for chozhang:
.
----------------------------------------------------------------------
Ran 1 test in 28.589s

OK
```

### CODE REVIEW

File: pulp_smash/tests/rpm/cli/test_sync.py [hunk 1]
```diff
@@ -99,3 +103,60 @@ def tearDownClass(cls):
...
+    .. _Pulp Smash #301: https://github.com/PulpQE/pulp-smash/issues/301
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Create and sync a repository."""
+        cls.cfg = config.get_config()
+        cls.repo_id = utils.uuid4()
+        cls.client = cli.Client(cls.cfg)
+        cls.sudo = '' if is_root(cls.cfg) else 'sudo '
```
Comment (danuzclaudes) [inline pulp_smash/tests/rpm/cli/test_sync.py]: I'm not sure if executing sudo will require password, thus messing up automation?

Comment (Ichimonji10) [inline pulp_smash/tests/rpm/cli/test_sync.py]: Pulp Smash must be able to log into the remote system either as root or as a user that can execute `sudo ...` without a password. It's the user's job to make that happen.

**Comment by elyezer** [comment]
Merged as cd814fef295b3c8d6215f6f654c239551ebb199a with a follow up commit of 72426654cbd1b45620eff03a83cf4ae75fadd931

**Comment by danuzclaudes** [comment]
@elyezer: Thank you again for reviewing it!

**Pull request closed without merging**